fineract-progressive-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/transactionprocessor/impl/ProgressiveTransactionCtx.java

```diff
@@ -45,22 +46,27 @@ public class ProgressiveTransactionCtx extends TransactionCtx {
     @Setter
     private boolean isWrittenOff = false;
     @Setter
+    private LoanReAgeInterestHandlingType reAgeInterestHandlingType;
```

Comment:
i dont think we need this. The reage transaction has all the required reage parameters. It should not be stored in the context as multiple reage txn can be on a loan.